Modules/booking/Model/BkCalendarEntry.php

```diff
@@ -546,6 +555,9 @@ public function getEntriesForPeriodeAndResources($id_space, $dateBegin, $dateEnd
             if(!$data[$i]["color_text"]) {
                 $data[$i]["color_text"] = "#000000";
             }
+            if(!$data[$i]["client_name"]) {
+                $data[$i]["client_name"] = "no client defined";
```

Comment:
as it is a model fonction, if no client name defined, should not be replaced by a string here but remain as null (or empty).
it is in view that empty/null name should be replaced by a default string

Reply:
Fixed. I found a bug. Not sure it is linked to this PR. I'm checking